### PR TITLE
[npmi] View switching

### PR DESCRIPTION
**Pull request by Sparkier**
* Motivation for features / changes
Before adding the embeddings visualization, one needs to be able to switch between those views.

* Technical description of changes
Can now switch between embeddings and default view.

* Screenshots of UI changes
image

* Detailed steps to verify changes work correctly (as executed by you)
`bazel test //tensorboard/webapp:karma_test_chromium_local`
`bazel run tensorboard -- --logdir [your logdir]`
Go to http://localhost:6007/?experimentalPlugin=npmi

### CODE REVIEW

**Comment by Sparkier** [comment]
@stephanwlee, I cannot mirror this:
```
The following URLs are not properly mirrored:
  - http://mirror.tensorflow.org/raw.githubusercontent.com/google/material-design-icons/3.0.1/action/svg/production/ic_group_work_24px.svg
Please comment on your PR asking a TensorBoard core team member 
to mirror these URLs per instructions in http://b/133880558.
```
Can you do that for me?

File: tensorboard/webapp/plugins/npmi/actions/npmi_actions.ts [hunk 1]
```diff
@@ -122,7 +122,21 @@ export const npmiShowHiddenAnnotationsToggled = createAction(
   '[NPMI] Show Hidden Annotations Toggled'
 );
 
+export const npmiToggleEmbeddingsView = createAction(
```
Comment (stephanwlee) [inline tensorboard/webapp/plugins/npmi/actions/npmi_actions.ts:125]: Prefer past tense for these action names, i.e., `npmiEmbeddingsViewToggled`. Here and below.

File: tensorboard/webapp/plugins/npmi/views/data_selection/metric_arithmetic/metric_arithmetic_element/metric_arithmetic_element_component.ng.html [hunk 2]
```diff
@@ -15,6 +15,13 @@
 limitations under the License.
 -->
 <mat-chip class="filter-chip" (removed)="onRemove.emit(metric)">
+  <mat-icon
+    *ngIf="embeddingsPresent"
+    class="embeddings-button"
+    svgIcon="group_work_24px"
+    (click)="onSelect.emit(metric)"
+    [ngClass]="{'embedding-selected': embeddingsMetric===metric, 'embedding-unselected': embeddingsMetric!==metric}"
```
Comment (stephanwlee) [inline tensorboard/webapp/plugins/npmi/views/data_selection/metric_arithmetic/metric_arithmetic_element/metric_arithmetic_element_component.ng.html:23]: `embeddingsMetric!==metric` -> `embeddingsMetric !== metric`

File: tensorboard/webapp/plugins/npmi/views/data_selection/metric_arithmetic/metric_arithmetic_element/metric_arithmetic_element_container.ts [hunk 3]
```diff
@@ -53,13 +60,23 @@ export class MetricArithmeticElementContainer {
       return {min: min, max: max};
     })
   );
+  readonly embeddingsPresent$ = this.store.pipe(select(getEmbeddingData)).pipe(
```
Comment (stephanwlee) [inline tensorboard/webapp/plugins/npmi/views/data_selection/metric_arithmetic/metric_arithmetic_element/metric_arithmetic_element_container.ts:63]: `embeddingsPresent` -> `embeddingsDataPresent` (or `hasEmbeddingsData`).

**Comment by stephanwlee** [comment]
Updated the mirror to include the group_work_24px.svg.

**Review by stephanwlee — APPROVED** [review]

**Pull request merged**